### DISCUSSION TITLE
Problem with Tango 2 Pro - Happymodel ES24TX Slim Pro

### DISCUSSION THREAD

**Discussion by hflopez** [ExpressLRS/ExpressLRS#1180] [General]
Hi everyone, 
I have a brand new Tango 2  V3 pro
I bought the multiprotocol bay adapter and installed it

Updated the elrs slim pro module to the new 2.0.0 elrs firmware (HM ES24TX Pro Series 2400 TX),
Tried with 1.2.1 firmware  version before and had the same problem.

Connected the slim pro elrs adapter, and it powers up.

When trying to execute the ELRS lua script (version 2 in this case) it gets stucked in the loading page.
Looks like if there was a comunication problem between the module and the mainboard of the transmitter, but i have checked many times the flat cable between them and it is ok...

Don´t know if it is a problem with my mainboard version.
It says VG07S

Hope someone can help me :)

![photo_2021-12-13_14-37-16](https://user-images.githubusercontent.com/96062585/145822970-82facd68-fbf6-442d-8ad3-87a82159c117.jpg)

**Comment by Shynrore** [comment]
Hello, I have the same problem with the tango 2 and happy model TX nano pro 2.4ghz because I put the Lua script elrs v2 on my tango, when I enter in the Lua script it freeze my tango 2 with loading       ???  But when I change the Lua script for 1.1.0 it work for me. Maybe it can help you.

**Reply by hflopez** [reply]
thanks for the reply!
so you used lua script v1 and module firmware 1.1.0? 
I think i tried all possible versions, but will check again...

**Reply by Shynrore** [reply]
Yes firmware 1.1.0 and Lua scriot 1.1.0, I can enter the menu of the Lua script with no freezing my tango 2 pcb v3 then if it work for you with 1.1.0 you can maybe upgrade  later for the v2 lua. Hope it work for you. Best luck.
